scripts/lib/setup_path_on_import.py

```diff
@@ -5,14 +5,11 @@
 import os
 import sys
 
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
-
-venv = os.path.join(BASE_DIR, "zulip-py3-venv")
-if sys.prefix != venv:
+if os.path.basename(sys.prefix) != "zulip-py3-venv":
```

Comment:
Can you explain the reasoning round making this just a basename check?

Reply:
I was still considering options here, but the relevant issues are

1. We might have activated the virtualenv through a different symlink, in which case there’s no need to reactivate it.
2. The `BASE_DIR/zulip-py3-venv` symlink still isn’t present during `build-release-tarball`. (Should it be?)

Reply:
I think if we're trying to have it be available everywhere, then yes, it should be.